projects/govuk_publishing_components/Makefile

```diff
@@ -1,2 +1,2 @@
 govuk_publishing_components: bundle-govuk_publishing_components
-	$(GOVUK_DOCKER) run $@-lite npm install
+	$(GOVUK_DOCKER) run $@-lite yarn install
```

Comment:
Don't we also need to run `npm install` or is that covered by yarn?

Reply:
It's covered by yarn. Yarn is essentially a drop in replacement for npm.